### PR TITLE
feat: add api helper

### PR DESCRIPTION
**Pull request by julianiff**
- renames api call and adds helpers for all api calls

### CODE REVIEW

**Review by marguslant0 — APPROVED** [review]
:thu

File: packages/app-bridge/src/registries/api/CreateAsset.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (julianiff) [inline packages/app-bridge/src/registries/api/CreateAsset.spec.ts:3]:
```suggestion
```

File: packages/app-bridge/src/registries/api/GetAssetResourceInformation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (julianiff) [inline packages/app-bridge/src/registries/api/GetAssetResourceInformation.spec.ts:3]:
```suggestion
```

File: packages/app-bridge/src/registries/api/GetCurrentUser.spec.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```
Comment (julianiff) [inline packages/app-bridge/src/registries/api/GetCurrentUser.spec.ts:3]:
```suggestion
```

**Pull request merged**